src/dbhelper.py

```diff
@@ -0,0 +1,107 @@
+from PySide import QtSql
+from PySide.QtSql import QSqlQuery
+from passwords import DB_SERVER, DB_PORT, DB_LOGIN, DB_PASSWORD, DB_TABLE
+import logging
+
+# Object to wrap a database connection and provide some vaguely handy utilities.
+_logger = logging.getLogger(__name__)
+
+# Open the database connection. when the module is first touched...
+_db = QtSql.QSqlDatabase("QMYSQL")
+_db.setHostName(DB_SERVER)
+_db.setPort(DB_PORT)
```

Comment:
While it's handy to open the connection at module-import time, it is a bit problematic in some cases:

 - When running tests, we don't want to contact the database, and ensuring that this code doesn't run becomes a problematic manner of managing every `import` statement in every file imported from tests, to ensure that you monkeypatch this module first.
 - When compiling the code with e.g. nuitka, it will try to run this code at compile-time.

It would be preferable to establish the connection on the first query, instead.